blockchains/smartchain/assets/0x7A2172F3728962F6B952a1Cc1AD794DACbf9f3DA/info.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "VIDTC Company stocks",
+    "website": "https://about.v-id.org",
+    "short_description": "NFTs for equity and shared ownership",
+    "explorer": "https://bscscan.com/address/0x7a2172f3728962f6b952a1cc1ad794dacbf9f3da"
```

Comment:
Update this as well to `https://bscscan.com/token/0x7a2172f3728962f6b952a1cc1ad794dacbf9f3da`